prisma/schema.prisma

```diff
@@ -25,6 +25,7 @@ enum AuthType {
   PASSWORD
   OAUTH
   GITHUB
```

Comment:
we do not need GOOGLE it is just OAUTH.  just use that.  we do not need a new enum for each approving site.